solr/core/src/test/org/apache/solr/cloud/AliasIntegrationTest.java

```diff
@@ -99,17 +99,6 @@ public void testProperties() throws Exception {
     CollectionAdminRequest.createCollection("collection2meta", "conf", 1, 1)
         .process(cluster.getSolrClient());
 
-    cluster.waitForActiveCollection("collection1meta", 2, 2);
-    cluster.waitForActiveCollection("collection2meta", 1, 1);
-
-    waitForState(
-        "Expected collection1 to be created with 2 shards and 1 replica",
-        "collection1meta",
-        clusterShape(2, 2));
-    waitForState(
-        "Expected collection2 to be created with 1 shard and 1 replica",
-        "collection2meta",
-        clusterShape(1, 1));
```

Comment:
The point of this issue/PR is to be able to use a CloudSolrClient (such as cluster.getSolrClient) after collection creation to make requests to the collection (index/query) without explicitly waiting for its existence.  Nothing more.  This will enable *some* tests to remove waitForState calls that follow collection creation as needless.  Maybe a comment would be nice on waitForState to elaborate on when it might *not* be needed.

Here where I comment, I don't see that we can remove these calls.  If you've thought about this one and think otherwise, let me know.  At this time I don't plan to review the remainder of the tests; you may need to revert some.  And to reduce test flapping risk, I would prefer that these waitForState removal changes are decoupled to another PR that is merged separately.  We might intentionally be delay the 9x backport to make changes here more apparent.  This would also be good for code archeology to see the substance of the change without distraction of touching 30+ other files.

Reply:
Got it and I agree - will revert the last commit.